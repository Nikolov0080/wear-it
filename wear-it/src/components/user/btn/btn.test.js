import React from 'react';
import renderer from 'react-test-renderer';
import AuthButton from './btn';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Auth Button', () => {
    const tree = renderer
        .create(<BrowserRouter><AuthButton  /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});