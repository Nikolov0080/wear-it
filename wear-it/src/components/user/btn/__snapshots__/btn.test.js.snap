// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Auth Button 1`] = `
<div>
  <button
    className="btn"
    type="submit"
  />
</div>
`;
